src/Resource.php

```diff
@@ -176,14 +176,6 @@ public function eloquentRelationships(): Collection
             ->mapWithKeys(function (Field $field) {
                 $eloquentRelationship = $field->handle();
 
-                // If the field has a `relationship_name` config key, use that instead.
-                // Sometimes a column name will be different to the relationship name
-                // (the method on the model) so our magic won't be able to figure out what's what.
-                // Eg. standard_parent_id -> parent
-                if ($field->get('relationship_name')) {
-                    $eloquentRelationship = $field->get('relationship_name');
-                }
-
```

Comment:
Instead of moving this code down below, let's just do an early return here instead:

```php
return [$field->handle() => $field->get('relationship_name')];
```

Reply:
Yep - done.

Reply:
Thanks! I'll get this merged & tagged next week. 👍